docs/website/docs/walkthroughs/deploy-a-pipeline/deploy-with-dagster.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Deploy with Dagster
+description: How to deploy a pipeline with Dagster
+keywords: [how to, deploy a pipeline, Dagster]
+---
+
+# Deploy with Dagster
+
+
+## Introduction to Dagster
+
+Dagster is an orchestrator that's designed for developing and maintaining data assets, such as tables, data sets, machine learning models, and reports. Dagster ensures these processes are reliable and focuses on using software-defined assets (SDAs) to simplify complex data management, enhance the ability to reuse code and provide a better understanding of data.
+
+To read more, please refer to Dagster’s [documentation.](https://docs.dagster.io/getting-started?_gl=1*19ikq9*_ga*NTMwNTUxNDAzLjE3MDg5Mjc4OTk.*_ga_84VRQZG7TV*MTcwOTkwNDY3MS4zLjEuMTcwOTkwNTYzNi41Ny4wLjA.*_gcl_au*OTM3OTU1ODMwLjE3MDg5Mjc5MDA.)
+
+### Dagster Cloud Features
+
+Dagster Cloud further enhances these features by providing an enterprise-level orchestration service with serverless or hybrid deployment options. It incorporates native branching and built-in CI/CD to prioritize the developer experience. It enables scalable, cost-effective operations without the hassle of infrastructure management. 
+
+### Dagster deployment options: **Serverless** versus **Hybrid**:
+
+The *serverless* option fully hosts the orchestration engine, while the *hybrid* model offers flexibility to use your computing resources, with Dagster managing the control plane. Reducing operational overhead and ensuring security.
+
+For more info, please [refer.](https://dagster.io/cloud)
+
+### Using Dagster for Free
+
+Dagster offers a 30-day free trial during which you can explore its features, such as pipeline orchestration, data quality checks, and embedded ELTs. You can try Dagster using its open source or by signing up for the trial. 
+
+## Building Data Pipelines with `dlt`
+
+`dlt` is an open-source Python library that allows you to declaratively load data sources into well-structured tables or datasets through automatic schema inference and evolution. It simplifies building data pipelines by providing functionality to support the entire extract and load process.
+
+How does `dlt` integrate with Dagster for pipeline orchestration?
+
+`dlt` integrates with Dagster for pipeline orchestration, providing a streamlined process for building, enhancing, and managing data pipelines. This enables developers to leverage `dlt`'s capabilities for handling data extraction and load and Dagster's orchestration features to efficiently manage and monitor data pipelines.
+
+Here’s a brief summary of how to orchestrate `dlt` pipeline on Dagster:
+
+1. Create a `dlt` pipeline. For detailed instructions on creating a pipeline, please refer to the 
+[documentation](https://dlthub.com/docs/walkthroughs/create-a-pipeline).
+
+1. Set up a Dagster project, configure resources, and define the asset. For more information, please refer to [Dagster’s documentation.](https://docs.dagster.io/getting-started/quickstart)
```

Comment:
You can use the following for Dagster:

-  create a Dagster Project: 
```
mkdir dagster_github_issues
cd dagster_github_issues
dagster project scaffold --name github-issues
```

- Define dlt as a dagster resource:
```
from dagster import ConfigurableResource
import dlt

class DltResource(ConfigurableResource):
    pipeline_name: str
    dataset_name: str
    destination: str

    def create_pipeline(self, resource_data, table_name):

        # configure the pipeline with your destination details
        pipeline = dlt.pipeline(
        pipeline_name=self.pipeline_name, destination=self.destination, dataset_name=self.dataset_name
        )

        # run the pipeline with your parameters
        load_info = pipeline.run(resource_data, table_name=table_name)

        return load_info
```

- Define Asset:
```
@asset
def issues_pipeline(pipeline: DltResource):

    logger = get_dagster_logger()
    results = pipeline.create_pipeline(github_issues_resource, table_name='github_issues')
    logger.info(results)
```

- run the webserver:
`dagster dev`

Reply:
added